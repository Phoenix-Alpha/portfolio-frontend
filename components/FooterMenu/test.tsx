import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { FooterMenu } from './index'

describe('<FooterMenu />', () => {
  it('should renders correct html', () => {
    const menu = [
      {
        key: 'test-about-key',
        title: 'About',
        url: '/about'
      },
      {
        key: 'test-example-key',
        title: 'Example',
        url: 'https://example.com'
      }
    ]

    const wrapper = shallow(<FooterMenu menu={menu} />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
