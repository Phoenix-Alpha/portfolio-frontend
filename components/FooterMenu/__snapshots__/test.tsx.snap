// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FooterMenu /> should renders correct html 1`] = `
<div
  className="flex flex-row justify-around items-center text-sm font-normal lg:space-x-[60px] leading-[22px] max-w-[472px] w-full text-secondary"
>
  <div
    key="About"
  >
    <Link
      href="/about"
    >
      <a
        className="hover:text-primary"
      >
        About
      </a>
    </Link>
  </div>
  <div
    key="Example"
  >
    <Link
      href="https://example.com"
    >
      <a
        className="hover:text-primary"
      >
        Example
      </a>
    </Link>
  </div>
</div>
`;
